src/Sylius/Component/Grid/spec/DataExtractor/PropertyAccessDataExtractorSpec.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Component\Grid\DataExtractor;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Sylius\Component\Grid\DataExtractor\DataExtractorInterface;
+use Sylius\Component\Grid\DataExtractor\PropertyAccessDataExtractor;
+use Sylius\Component\Grid\Definition\Field;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+
+/**
+ * @mixin PropertyAccessDataExtractor
+ *
+ * @author Paweł Jędrzejewski <pawel@sylius.org>
+ */
+class PropertyAccessDataExtractorSpec extends ObjectBehavior
+{
+    function let(PropertyAccessorInterface $propertyAccessor)
+    {
+        $this->beConstructedWith($propertyAccessor);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Component\Grid\DataExtractor\PropertyAccessDataExtractor');
+    }
+
+    function it_is_a_data_extractor()
```

Comment:
I mean that it should be `it_is_*` rather than `it_implements_*` everywhere, it is more natural, human-readable syntax, that tells us what it is instead of what are the implementation details.